src/SUMMARY.md

```diff
@@ -1,11 +1,12 @@
 # Rust Forge
 
 [Overview](./README.md)
-- [Chat Platforms](./chat/README.md)
+- [Platforms](./chat/README.md)
```

Comment:
[As per suggestion](https://rust-lang.zulipchat.com/#narrow/stream/224082-t-release.2Ftriagebot/topic/Triagebot.20documentation/near/201731148) from @XAMPPRocky

Reply:
Thanks for doing this, would you also be able to add redirects for the existing pages? It just needs to be added to the `book.toml`. https://github.com/rust-lang/rust-forge/blob/236e793960d900ea97901a88048acbac0238bf7a/book.toml#L17-L25